packages/automerge-repo/src/synchronizer/DocSynchronizer.ts

```diff
@@ -2,10 +2,12 @@ import EventEmitter from "eventemitter3"
 import { Synchronizer, SyncMessages } from "./Synchronizer"
 import * as Automerge from "@automerge/automerge"
 import { DocHandle, DocumentId } from "../DocHandle"
-import { ChannelId, PeerId } from "../network/NetworkSubsystem"
+import { ChannelId, DecodedMessage, PeerId } from "../network/NetworkSubsystem"
 import debug from "debug"
+import { decodeChange, DecodedSyncMessage } from "@automerge/automerge"
 const log = debug("DocSynchronizer")
 const conciseLog = debug("DocSynchronizerConcise") // Only logs one line per receive/send
+const opsLog = debug("DocSynchronizerOps") // Log list of ops of each message
```

Comment:
this (and above) can use a colon `DocSynchronizer:Ops` to allow more targeted logging